src/ngx_lua_resty_lmdb_module.c

```diff
@@ -339,6 +337,121 @@ static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle)
 }
 
 
+static ngx_int_t
+ngx_lua_resty_lmdb_close_file(ngx_cycle_t *cycle,
+                              ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    mdb_txn_abort(lcf->ro_txn);
+    mdb_env_close(lcf->env);
+
+    return NGX_OK;
+}
+
+
+static ngx_int_t
+ngx_lua_resty_lmdb_verify_file_status(ngx_cycle_t *cycle,
+                                      ngx_lua_resty_lmdb_conf_t *lcf)
+{
+    ngx_core_conf_t  *ccf;
+    ngx_file_info_t   fi;
+
+    size_t            max_name_len;
+    u_char           *name_buf;
+    ngx_str_t        *name;
+    ngx_str_t         names[] = { ngx_string("/"),
+                                  ngx_string("/data.mdb"),
+                                  ngx_string("/lock.mdb"),
+                                  ngx_null_string,
+                                };
+
+    ccf = (ngx_core_conf_t *) ngx_get_conf(cycle->conf_ctx, ngx_core_module);
+
+    if (ccf->user == (ngx_uid_t) NGX_CONF_UNSET_UINT) {
+        return NGX_OK;
+    }
+
+    max_name_len = lcf->env_path->name.len + 32;
+
+    name_buf = ngx_pnalloc(cycle->pool, max_name_len);
```

Comment:
This can be statically allocated right? We can do `u_char [PATH_MAX]` instead.

Reply:
done.